### PR TITLE
Support for orgData in the nomination list

### PR DESCRIPTION
**Pull request by twsvaishali**

### CODE REVIEW

File: src/service/programService.js [hunk 1]
```diff
@@ -533,7 +553,7 @@ function getNominationsList(req, response) {
 }
 
 function getUsersDetails(req, userId) {
-  const url = `${envVariables.baseURL}/content/reg/search`;
+  const url = `http://dock.sunbirded.org/api/reg/search`;
```
Comment (venkateshwarans) [inline src/service/programService.js:556]: Change it to env `baseURL`

File: src/service/programService.js [hunk 2]
```diff
@@ -560,6 +580,33 @@ function getUsersDetails(req, userId) {
   });
 }
 
+function getOrgDetails(req, orgId) {
+  const url = `http://dock.sunbirded.org/api/reg/search`;
```
Comment (venkateshwarans) [inline src/service/programService.js:584]: Change it to env `baseURL`

File: src/service/programService.js [hunk 3]
```diff
@@ -489,12 +494,27 @@ function getNominationsList(req, response) {
...
+
+        forkJoin(...userMap, ...orgMap).subscribe(([resData, orgData]) => {
+
           _.forEach(resData, function (data, index) {
             if (data.data.result) {
               result[index].dataValues.userData = data.data.result.User[0];
             }
           });
+
+          if (_.isEmpty(orgList)) {
```
Comment (venkateshwarans) [inline src/service/programService.js:510]: is it `orgList` empty or not empty?

File: src/service/programService.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (twsvaishali) [reply src/service/programService.js]: This was alredy done.. I think you check files from commit

**Pull request merged**